part-spec/common/unit.json

```diff
@@ -47,7 +47,8 @@
                     "megahertz",
                     "percentage",
                     "bit",
-                    "byte"
+                    "byte",
+                    "mega transfers per second"
```

Comment:
This is the only unit that has spaces... and it seems weird to have spaces in a unit. (e.g. it makes it harder to parse by other tooling later)

I am guessing this unit is already used elsewhere, so maybe a lost cause, or maybe just intentional.

Reply:
I actually just removed this from unit and changed it to "number". I think it's kind of weird here on second thought